src/Web3/Eth/Decoders.elm

```diff
@@ -38,15 +46,106 @@ blockDecoder =
         |> required "uncles" (list string)
 
 
+blockTxObjDecoder : Decoder BlockTxObjs
```

Comment:
Need to go through each decoder thoroughly and handle all the potential `nullable` and `optional`'s.

We want elm-web3 to work with MainNet, TestRpc, Ropsten, Kovan, etc. With and without MetaMask.